app/src/main/res/drawable/ic_qrcode_scan.xml

```diff
@@ -4,6 +4,6 @@
         android:viewportWidth="24.0"
         android:viewportHeight="24.0">
     <path
-        android:fillColor="@color/gray_dark"
+        android:fillColor="@color/primary"
```

Comment:
Is this `drawable` getting used anywhere? Please check

Reply:
Yes, This this will be the main coloured qr code.

Reply:
The other QR code is used at two places.